### PR TITLE
remove spec reporter, try and run

### PR DESCRIPTION
**Pull request by CBonade**
## Description
This PR removes the duplicate output of the Cypress test spec results as well as removes the caching that was causing Cypress to break our builds.  Also adds one new test spec.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by holdenhinkle — APPROVED** [review]
I'm not up to speed on the caching issue so I can't provide a review for that. But the rest looks good.

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -144,7 +144,7 @@ jobs:
         with:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-region: "us-gov-west-1"
+          aws-region: 'us-gov-west-1'
```
Comment (U-DON) [inline .github/workflows/continuous-integration.yml:147]: Actually might not need any quotes at all.

**Review by ddzz — APPROVED** [review]

File: src/applications/static-pages/tests/application-status.cypress.skip.spec.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      cy.wrap(button).click();
+    });
+  cy.get('#start-over-modal-title', { timeout: Timeouts.normal }).should(
+    'contain',
+    'Starting over will delete your in-progress application.',
+  );
+  cy.axeCheck();
+});
+
+describe('Application Status Test', () => {
```
Comment (U-DON) [inline src/applications/static-pages/tests/application-status.cypress.skip.spec.js:26]: Do y'all care about counting skipped tests in your data? If so, could use `describe.skip` here.

**Review by U-DON — APPROVED** [review]
This is fine, but the test should probably be in a separate PR for cleaner separation. Then this could just be focused on Cypress GHA improvements.

File: src/applications/static-pages/tests/application-status.cypress.skip.spec.js (hunk 2 above)
Comment (CBonade) [reply src/applications/static-pages/tests/application-status.cypress.skip.spec.js:26]: it's not going to be permanent. Honestly, this wound up in here by mistake. It's for a future batch of tests that I'm currently working on converting from nightwatch.  I figured I'd let it skip for now, and then when it's done it'll end up counted properly.

Comment (CBonade) [reply src/applications/static-pages/tests/application-status.cypress.skip.spec.js]: removed it from the branch on this PR

File: .github/workflows/continuous-integration.yml (hunk 1 above)
Comment (CBonade) [reply .github/workflows/continuous-integration.yml]: removed these quotes entirely

File: src/applications/static-pages/tests/application-status.cypress.spec.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    'contain',
+    'Starting over will delete your in-progress application.',
+  );
+  cy.axeCheck();
+});
+
+describe('Application Status Test', () => {
+  it('Achieves the correct result per URL', () => {
+    cy.login();
+    /* eslint-disable camelcase */
```
[the remark this answers is not in the document]
Comment (CBonade) [reply src/applications/static-pages/tests/application-status.cypress.spec.js]: no longer relevant as this file has been removed from this PR

File: src/applications/static-pages/tests/application-status.cypress.skip.spec.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  it('Achieves the correct result per URL', () => {
+    cy.login();
+    /* eslint-disable camelcase */
```
[the remark this answers is not in the document]
Comment (CBonade) [reply src/applications/static-pages/tests/application-status.cypress.skip.spec.js]: no longer relevant as this file has been removed from this PR

File: .github/workflows/continuous-integration.yml [hunk 5]
```diff
@@ -324,19 +324,20 @@ jobs:
...
 
-      - name: Cache dependencies
-        id: cache-dependencies
-        uses: actions/cache@v2
-        with:
-          path: |
-            ~/.cache/yarn
-            node_modules
-          key: ${{ steps.get-node-version.outputs.NODE_VERSION }}-${{ hashFiles('yarn.lock') }}
+      # Disabling caching due to cypress breaking - to be evaluated at a later date
```
Comment (oseasmoran73) [inline .github/workflows/continuous-integration.yml:327]:
```suggestion
      # TODO: Disabling caching due to cypress breaking - to be evaluated at a later date
```

**Review by oseasmoran73 — APPROVED** [review]

**Pull request merged**